### PR TITLE
fromString: Choose to search on one language

### PR DESCRIPTION
**Pull request by MarkGotham**

### CODE REVIEW

**Review by jacobtylerwalls — COMMENTED** [review]
Thanks Mark, I left mostly trivial feedback, but for ...

I think the biggest item to get a decision on would be whether the language names should be enums. Myke has mentioned elsewhere his desire to have only enums and not string configurations going forward. (For me, the biggest benefit there is to help with code completion in IDEs, etc.) I bet if the languages were already an enum, I would have already boldly clicked merge :D

File: music21/instrument.py [hunk 1]
```diff
@@ -758,6 +760,7 @@ def __init__(self):
         self.instrumentSound = 'pluck.koto'
         self.midiProgram = 107
 
+
```
[the remark this answers is not in the document]
Comment (MarkGotham) [reply music21/instrument.py:763]: pycodestyle.py tool, checking 'for violations of the PEP 8 coding style guide.'

**Comment by MarkGotham** [comment]
Thanks @jacobtylerwalls. How's this? It's a shame to be case-sensitive. Is there a way around that?

**Comment by jacobtylerwalls** [comment]
Yeah, I'm going to open a quick PR. Let's see if Myke is on board.

File: music21/instrument.py [hunk 2]
```diff
@@ -2382,42 +2405,69 @@ def fromString(instrumentString):
...
     '''
     # pylint: disable=undefined-variable
     from music21.languageExcerpts import instrumentLookup
 
+    if not language:
+        sourceDict = instrumentLookup.allToClassName
+    else:
+        if language not in SupportedLanguages:
+            raise InstrumentException(f'Chosen language {language} not currently supported.')
+        sourceDict = getattr(instrumentLookup, language + 'ToClassName')
```
Comment (jacobtylerwalls) [inline music21/instrument.py:2439]:
```suggestion
    language = language.lower()
    if language not in SearchLanguage:
        raise InstrumentException(f'Chosen language {language} not currently supported.')
    sourceDict = getattr(instrumentLookup, language + 'ToClassName')
```

**Review by jacobtylerwalls — COMMENTED** [review]
This should do it! (docs might need tidying up to correspond...)

**Pull request merged**